docs/lang/articles/basic/layout.md

```diff
@@ -58,13 +58,40 @@ ti.root.dense(ti.ij, (3, 4)).place(x)
 x = ti.field(ti.f32, shape=(3, 4))
 ```
 
-You can also nest two 1D `dense` statements to describe the same 2D array.
+You can also nest two 1D `dense` statements to describe a 2D array of the same shape.
 
 ```python {1-2}
 x = ti.field(ti.f32)
 ti.root.dense(ti.i, 3).dense(ti.j, 4).place(x)
+# has the same shape with
+x = ti.field(ti.f32, shape=(3,4))
 ```
 
+:::note
+
+The above 2D array built with nested `dense` statements is **NOT** equivalent to the 2D array built with `ti.field`.
+Although those statements both result in a 2D array of the same shape, they have
+different layers of `SNodeTree`. Namely,
+```python
+x = ti.field(ti.f32)
+ti.root.dense(ti.i, 3).dense(ti.j, 4).place(x)
+```
+has two layers of `SNodeTree` below the root, while
+```python
+x = ti.field(ti.f32)
+ti.root.dense(ti.ij, (3, 4)).place(x)
+# or equivalently
+x = ti.field(ti.f32, shape=(3,4))
+```
+only has one layer of `SNodeTree` below the root. See the sketch below:
+
+<img width="683" alt="Screen Shot 2022-09-16 at 10 38 39" src="https://user-images.githubusercontent.com/2747993/190545525-305563dc-d09e-4af2-b99b-166d5c4398d0.png">
```

Comment:
Docusaurus does not support this html tag. Use markdown instead.

Reply:
OK! Please see my latest commit.